framework/format/platform_types.h

```diff
@@ -189,24 +189,24 @@ extern "C"
     typedef VkResult(VKAPI_PTR* PFN_vkGetMemoryAndroidHardwareBufferANDROID)(
         VkDevice device, const VkMemoryGetAndroidHardwareBufferInfoANDROID* pInfo, struct AHardwareBuffer** pBuffer);
 
-    static VKAPI_ATTR VkResult VKAPI_CALL vkCreateAndroidSurfaceKHR(VkInstance,
-                                                                    const VkAndroidSurfaceCreateInfoKHR*,
-                                                                    const VkAllocationCallbacks*,
-                                                                    VkSurfaceKHR*)
+    inline static VKAPI_ATTR VkResult VKAPI_CALL vkCreateAndroidSurfaceKHR(VkInstance,
+                                                                           const VkAndroidSurfaceCreateInfoKHR*,
+                                                                           const VkAllocationCallbacks*,
+                                                                           VkSurfaceKHR*)
     {
         GFXRECON_LOG_ERROR("Calling unsupported platform extension function vkCreateAndroidSurfaceKHR");
         return VK_ERROR_EXTENSION_NOT_PRESENT;
     }
 
-    static VKAPI_ATTR VkResult VKAPI_CALL vkGetAndroidHardwareBufferPropertiesANDROID(
+    inline static VKAPI_ATTR VkResult VKAPI_CALL vkGetAndroidHardwareBufferPropertiesANDROID(
```

Comment:
`inline` rather than `inline static`?
There are loads of these in this file. I won't label them all.